docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,181 @@
 
-### AdonisFX Paint Tool
+- **Soft Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant distance to the closest point on the reference mesh at initialization. Painting these constraint weights would allow the deformer to create a wrinkle effect when combined with hard and slide weights.
 
-To configure the paintable attributes in the skin deformer, the AdonisFX paint tool must be used. Apart from the standard functionalities that the Maya default paint context provides, this tool also processes the painted weights to guarantee that the requirements of solver are satisfied.
+    - *Tip*: Flood the geometry with a very low value 0.1 - 0.2.
 
-<figure>
-  <img src="images/paint_tool.png" caption="AdonisFX Paint Tool"> 
-  <figcaption>Figure 2: AdonisFX Paint Tool</figcaption>
-</figure>
+    - *Tip*: To optimize the painting of the weight, flood it to 1.0 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
+
+- **Slide Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant distance to the reference mesh sliding along the reference surface. In the example of a biped or quadruped creature, it is recommended to set a value of 1.0 on the scapulas, shoulders, elbows and knees and an overall value of 0 on the rest of the body.
+
+    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure the simulation does not find hard edges as this could create undesired wrinkles.
+
+- **Compression Resistance**: Force to correct the edge lengths if the current length is smaller than the rest length. A higher value represents higher correction. At value 1 the points in the geometry will try to stay as close as possible to their original position.
+
+    - *Tip*: To optimize the painting of the weight, flood it to 1.0 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
+    
+    - *Tip*: Reducing the value of the weight in some areas will contribute to getting rid of unwanted wrinkles or possible artifacts in the skin.
 
-Do the following to open the tool:
+- **Stretching Resistance**: Force to correct the edge lengths if the current length is greater than the rest length. A higher value represents higher correction.
 
-  1. Select the mesh with the Skin deformer applied to.
-  2. Press the paint tool ![paint tool](images/adn_paint_tool.png) shelf button or go to AdonisFX menu > Paint Tool.
+    - *Tip*: To optimize the painting of the weight, flood it to 1.0 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
 
-You can select the attribute to paint from the combo box exposed in the UI:
+    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure the simulation doesn’t find hard edges as this could create undesired wrinkles.
```

Comment:
```suggestion
    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure the simulation does not find hard edges as this could create undesired wrinkles.
```

Reply:
Applied!